cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -53,7 +53,7 @@ public void testCreateUserTask() {
     Assert.assertEquals(userTaskHeader.getValue(), UserTaskManager.USER_TASK_HEADER_NAME);
     Assert.assertEquals(userTaskHeaderValue.getValue(), testUserTaskId.toString());
     Assert.assertEquals(future, future1);
-
+    EasyMock.verify(mockHttpServletResponse, mockHttpSession);
```

Comment:
Should this be `EasyMock.verify(mockUuidGenerator, mockHttpServletResponse, mockHttpSession);`?